source/wp-content/themes/wporg-news-2021/sass/base/_normalize.scss

```diff
@@ -11,6 +11,11 @@ body {
 	-moz-osx-font-smoothing: grayscale;
 }
 
+// Blocks, by default, have a top margin. This is annoying.
+.wp-site-blocks [class*="wp-container-"] * + * {
+	margin-top: 0;
+}
```

Comment:
Annoying, but needed in the post content - right now the content is all collapsed against each other. I wonder if we can set `blockGap` on some of the containers that shouldn't have it.

Reply:
I added the `blockGap` back to the `post-content` block itself, with this reset applied everywhere else.